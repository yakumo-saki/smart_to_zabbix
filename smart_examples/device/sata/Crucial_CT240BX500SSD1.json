{
    "json_format_version": [
      1,
      0
    ],
    "smartctl": {
      "version": [
        7,
        2
      ],
      "svn_revision": "5155",
      "platform_info": "x86_64-linux-5.15.7-zen1-1-zen",
      "build_info": "(local build)",
      "argv": [
        "smartctl",
        "--json",
        "-a",
        "/dev/sdb"
      ],
      "exit_status": 32
    },
    "device": {
      "name": "/dev/sdb",
      "info_name": "/dev/sdb [SAT]",
      "type": "sat",
      "protocol": "ATA"
    },
    "model_family": "Crucial/Micron Client SSDs",
    "model_name": "CT240BX500SSD1",
    "serial_number": "************",
    "wwn": {
      "naa": 0,
      "oui": 0,
      "id": 0
    },
    "firmware_version": "M6CR013",
    "user_capacity": {
      "blocks": 468862128,
      "bytes": 240057409536
    },
    "logical_block_size": 512,
    "physical_block_size": 512,
    "rotation_rate": 0,
    "form_factor": {
      "ata_value": 3,
      "name": "2.5 inches"
    },
    "trim": {
      "supported": true,
      "deterministic": true,
      "zeroed": true
    },
    "in_smartctl_database": true,
    "ata_version": {
      "string": "ACS-2 T13/2015-D revision 3",
      "major_value": 1008,
      "minor_value": 272
    },
    "sata_version": {
      "string": "SATA 3.2",
      "value": 255
    },
    "interface_speed": {
      "max": {
        "sata_value": 14,
        "string": "6.0 Gb/s",
        "units_per_second": 60,
        "bits_per_unit": 100000000
      },
      "current": {
        "sata_value": 2,
        "string": "3.0 Gb/s",
        "units_per_second": 30,
        "bits_per_unit": 100000000
      }
    },
    "local_time": {
      "time_t": 1639372931,
      "asctime": "Mon Dec 13 14:22:11 2021 JST"
    },
    "smart_status": {
      "passed": true
    },
    "ata_smart_data": {
      "offline_data_collection": {
        "status": {
          "value": 0,
          "string": "was never started"
        },
        "completion_seconds": 120
      },
      "self_test": {
        "status": {
          "value": 0,
          "string": "completed without error",
          "passed": true
        },
        "polling_minutes": {
          "short": 2,
          "extended": 10
        }
      },
      "capabilities": {
        "values": [
          17,
          2
        ],
        "exec_offline_immediate_supported": true,
        "offline_is_aborted_upon_new_cmd": false,
        "offline_surface_scan_supported": false,
        "self_tests_supported": true,
        "conveyance_self_test_supported": false,
        "selective_self_test_supported": false,
        "attribute_autosave_enabled": false,
        "error_logging_supported": true,
        "gp_logging_supported": true
      }
    },
    "ata_smart_attributes": {
      "revision": 1,
      "table": [
        {
          "id": 1,
          "name": "Raw_Read_Error_Rate",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 47,
            "string": "POSR-K ",
            "prefailure": true,
            "updated_online": true,
            "performance": true,
            "error_rate": true,
            "event_count": false,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 5,
          "name": "Reallocate_NAND_Blk_Cnt",
          "value": 100,
          "worst": 100,
          "thresh": 10,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 9,
          "name": "Power_On_Hours",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 512,
            "string": "512"
          }
        },
        {
          "id": 12,
          "name": "Power_Cycle_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 838,
            "string": "838"
          }
        },
        {
          "id": 171,
          "name": "Program_Fail_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 172,
          "name": "Erase_Fail_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 173,
          "name": "Ave_Block-Erase_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 52,
            "string": "52"
          }
        },
        {
          "id": 174,
          "name": "Unexpect_Power_Loss_Ct",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 50,
            "string": "50"
          }
        },
        {
          "id": 180,
          "name": "Unused_Reserve_NAND_Blk",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 100,
            "string": "100"
          }
        },
        {
          "id": 183,
          "name": "SATA_Interfac_Downshift",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 184,
          "name": "Error_Correction_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 187,
          "name": "Reported_Uncorrect",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 194,
          "name": "Temperature_Celsius",
          "value": 54,
          "worst": 22,
          "thresh": 50,
          "when_failed": "past",
          "flags": {
            "value": 34,
            "string": "-O---K ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": false,
            "auto_keep": true
          },
          "raw": {
            "value": 335009087534,
            "string": "46 (Min/Max 25/78)"
          }
        },
        {
          "id": 196,
          "name": "Reallocated_Event_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 197,
          "name": "Current_Pending_ECC_Cnt",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 198,
          "name": "Offline_Uncorrectable",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 48,
            "string": "----CK ",
            "prefailure": false,
            "updated_online": false,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 199,
          "name": "UDMA_CRC_Error_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 202,
          "name": "Percent_Lifetime_Remain",
          "value": 97,
          "worst": 97,
          "thresh": 1,
          "when_failed": "",
          "flags": {
            "value": 48,
            "string": "----CK ",
            "prefailure": false,
            "updated_online": false,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 97,
            "string": "97"
          }
        },
        {
          "id": 206,
          "name": "Write_Error_Rate",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 46,
            "string": "-OSR-K ",
            "prefailure": false,
            "updated_online": true,
            "performance": true,
            "error_rate": true,
            "event_count": false,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 210,
          "name": "Success_RAIN_Recov_Cnt",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 0,
            "string": "0"
          }
        },
        {
          "id": 246,
          "name": "Total_LBAs_Written",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 5864799343,
            "string": "5864799343"
          }
        },
        {
          "id": 247,
          "name": "Host_Program_Page_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 183274979,
            "string": "183274979"
          }
        },
        {
          "id": 248,
          "name": "FTL_Program_Page_Count",
          "value": 100,
          "worst": 100,
          "thresh": 50,
          "when_failed": "",
          "flags": {
            "value": 50,
            "string": "-O--CK ",
            "prefailure": false,
            "updated_online": true,
            "performance": false,
            "error_rate": false,
            "event_count": true,
            "auto_keep": true
          },
          "raw": {
            "value": 105209856,
            "string": "105209856"
          }
        }
      ]
    },
    "power_on_time": {
      "hours": 512
    },
    "power_cycle_count": 838,
    "temperature": {
      "current": 46
    },
    "ata_smart_error_log": {
      "summary": {
        "revision": 1,
        "count": 0
      }
    },
    "ata_smart_self_test_log": {
      "standard": {
        "revision": 1,
        "count": 0
      }
    }
  }
  