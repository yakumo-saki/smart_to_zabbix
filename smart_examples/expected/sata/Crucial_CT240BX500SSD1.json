{
  "model": "CT240BX500SSD1",
  "type": "sat",
  "rotation_rate": 0,
  "protocol": "ATA",
  "serial_number": "************",
  "power_cycle": 838,
  "power_on_hours": 512,
  "temperature": 46,
  "ssd.bytes_written": null,
  "ssd.lifespan": null,
  "smart_status_passed": 1
}
